CHANGES.md

```diff
@@ -1,3 +1,25 @@
+## edge-18.12.1
+
+* CLI
+  * **New** `linkerd routes` command displays per-route stats for any resource
+  * **New** Service profiles are now supported for external authorities!
+  * **New** `linkerd routes --open-api` flag that generates a service profile
+  based on an OpenAPI specification (swagger) file
+* Web UI
+  * **New** Top Routes page, served at `/routes`
+  * **New** Route metrics are now available in the resource detail pages for
+  services with configured profiles
+  * **New** You can now create and download a service profile from the Web UI
+* Controller
+  * **Improved** Controller components are now prefixed with `linkerd-` to prevent name collisions with existing resources
+  * **New** The `-disable-h2-upgrade` flag has been added to control automatic HTTP/2 upgrading
```

Comment:
maybe write this as `**New** linkerd install --disable-h2-upgrade flag...` to give context that it belongs to install.